doc/API.md

```diff
@@ -338,8 +338,20 @@ disregarded for forward compatibility.
 
 #### Request
 
-| Field          | Type   | Description                                         |
-| -------------- | ------ | --------------------------------------------------- |
+| Field          | Type   | Description                                                          |
+| -------------- | ------ | -------------------------------------------------------------------- |
+| `status`       | array  | Array of [Spend status](#spend_status) |
+
+##### Spend status
+
+Please note that this status refers only to the Spend transaction, with regarding to the signatures and the broadcast status.
+You'll have to manually fetch the vaults statuses if you want to know, for example, if the vault was canceled or not.
+
+| Value           | Description                                                                                                                                                                              |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `non_final`     | The Spend transaction is not final, we are awaiting signatures either from managers or cosigners                                                                                         |
+| `pending`       | The transaction is not broadcasted to the Bitcoin network. Either we are awaiting for the Unvault timelock to mature, or the transaction was canceled and we can't broadcast it anymore. |
```

Comment:
> or the transaction was canceled and we can't broadcast it anymore.

I think this is wrong. If the Spend was once broadcasted we never mark it back to `broadcastable` in the DB?

Reply:
You're right. Added a test for this